Urban-Fun/View Controllers/OtherProfileViewController.m

```diff
@@ -0,0 +1,73 @@
+//
+//  OtherProfileViewController.m
+//  Urban-Fun
+//
+//  Created by Maize Booker on 7/26/22.
+//
+
+#import "OtherProfileViewController.h"
```

Comment:
Let's discuss about this new screen, I'm not quite sure why we want a different ViewController here, my 2cents is to reuse the same ProfileViewController and just let the ViewController show different UI based on the use case